server/api/categoryRouter.js

```diff
@@ -1,25 +1,23 @@
 /**
  * Router for item actions
  */
+const Categories = require('../db/models/category');
 const express = require('express');
 const mongoose = require('mongoose');
-const bodyParser = require('body-parser');
-
-const Categories = require('../db/models/category');
 
 const router = express();
-
+router.use(express.json());
```

Comment:
It appears this is equivalent to app.use(bodyParser.json()); which is also equal to app.use(bodyParser.urlencoded({ extended: true }));. Can we consider changing this in the other files too?

"Source": https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json

Reply:
ohh cool, i checked the link u sent, it said with latest release u can go back to using router.use(express.json()). what would be the reason for change?

Reply:
sorry i meant app.use(express.json())^. we have app.use(express.json()) and app.use(bodyParse.urlencoded...) in index.js. not sure but I don't think we need router.use(express.json()) in each router file?

Reply:
We need it in each router file to enable testing for each router, otherwise they won't run (I tried it both ways but maybe something has changed?) Also if we have express.json() we don't need the bodyParse one